src/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+
+{
+	"extends": "../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "./",
+		"outDir": "../dist",
+		"composite": true
+	},
+	"include": [
+		".",
+		"./**/*.d.ts"
+	]
+}
```

Comment:
@vladfrangu we'll replace both this and the `base` with the new tsconfig package, right?

Reply:
Ok, keep us posted 👍

Reply:
You can see it in https://github.com/apify/apify-client-js/pull/185 actually 😄